assets/js/entry/data-land.js

```diff
@@ -0,0 +1,35 @@
+import Vue from "vue";
+import FileManager from "@/vue/FileManager";
+import '/assets/css/file-manager.css'
```

Comment:
Also add: `xmlverbatim.css`

Reply:
Added it

Reply:
I thought we had the xml representation, so this CSS doesn't seem to be needed. We should discuss and double check.

Reply:
Looks like it is not needed 👍